test/jdk/java/net/vthread/BlockingSocketOps.java

```diff
@@ -185,8 +186,10 @@ void testSocketReadPeerClose2() throws Exception {
                 Socket s2 = connection.socket2();
 
                 // delayed abrupt close of s2
-                s2.setSoLinger(true, 0);
-                runAfterParkedAsync(s2::close);
+                runAfterParkedAsync(() -> {
+                    s2.setSoLinger(true, 0);
+                    s2.close();
+                });
```

Comment:
This is okay but shouldn't be necessary as the linger option was set before starting the thread to close the other end. The linger setting is per socket rather than per-thread so curious why the original was problematic on AIX.

Reply:
This test was my last barrier to integration before I cleaned up the code. Now it seems that it is passing unmodified. So, I've reverted the change. I may see this failure again, but as you mentioned the test shouldn't need to be changed.